doc/doc.sh

```diff
@@ -20,10 +20,11 @@ fi
 
 case $1 in
   prepare)
-    rm -rf framework
+    if [ -n "$CI" ]; then
+      rm -rf framework
```

Comment:
We need to rm the framework in netlify builds

Reply:
I used `REVIEW_ID` which is a Netlify default environment variable.

Reply:
I removed cache on documentation framework node_modules. Git clone errored since framework folder is not empty